### PR TITLE
[ADMIN] : Delete formation from formation page

### PR DESCRIPTION
**Pull request by assma8jaballah**

### CODE REVIEW

**Review by LaserHuy — APPROVED** [review]
Good jobs !

File: src/Model/FormationManager.php [hunk 1]
```diff
@@ -23,4 +23,12 @@ public function selectAllByDiscipline(int $id): array
 
         return $statement = $statement->fetchAll();
     }
+
+    //fonction pour supprimer une formation
+    public function delete(int $id): void
```
Comment (Benja-san) [inline src/Model/FormationManager.php:28]: you already have a delete method from your abstractManager parent class, use it to avoid code duplication

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/Model/FormationManager.php (hunk 1 above)
Comment (LaserHuy) [reply src/Model/FormationManager.php:28]: yeah right exactly

Comment (assma8jaballah) [reply src/Model/FormationManager.php]: ok, fixed

**Pull request merged**